src/measure_theory/bochner_integration.lean

```diff
@@ -37,13 +37,46 @@ Step 5: Define the Bochner integral on L1 functions by extending the integral on
 
 ## Main statements
 
-`section bintegral` : basic properties of `simple_func.bintegral` proved.
+• Basic properties of the Bochner integral.
 
-`section instances` : `l1.simple_func` forms a `normed_space`.
+• The Lebesgue dominated convergence theorem : `tendsto_integral_of_dominated_convergence`
 
-`section coe_to_l1` : `coe` from `l1.simple_func` to `l1` is a uniform and dense embedding.
+See `section properties` at the end of the file to find these properties.
```

Comment:
I would rather have the details here: the idea is that someone who only wants to use properties of the Bochner integral could find in this docstring the main statements he would need, without having to browse through the file and the construction of the integral.